tests/components/keymitt_ble/__init__.py

```diff
@@ -77,6 +78,7 @@ async def connect(self, init):
     async def disconnect(self):
         """Mock disconnect."""
 
+    @property
     async def is_connected(self):
```

Comment:
This should not be async:
```suggestion
    def is_connected(self):
```
That's what causing "your changes to the test". You get a "coroutine object" otherwise, not `False`.

Reply:
aaaaah thanks